localisation/00_zuwc_united_localisation.csv

```diff
@@ -2981,12 +2965,19 @@ GetHorses_horses;horses;;;;;;;;;;;;;x
 GetEluneTheLightCap_The_Light;The Light;;;;;;;;;;;;;x
 GetPronoun_Her;Her;;;;;;;;;;;;;x
 GetPronoun_Its;Its;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+satanists_recruit;Recruit to [From.Society.GetName];;;;;;;;;;;;;x
+satanists_recruit_desc;Attempt to find out whether �Y[Root.GetTitledFirstName]�! is open to the power of the occult. If so, you may invite [Root.GetHerHim] to become a member of [From.Society.GetName].;;;;;;;;;;;;;x
+EVTDESC_MNM_7022_A;We have seen your good work, [Root.GetSocietyRank] [Root.GetFirstName]\nThe recruit you sent us, [satanists_recruit.GetTitledFirstName], has now completed [satanists_recruit.GetHerHis] induction into [Root.Society.GetName]. I thank you for this contribution � it will not be forgotten.\n\nSigned, [From.GetSocietyRank] [From.GetFirstName];;;;;;;;;;;;;x
+EVTDESC_MNM_7022_B;We know what has happened, [Root.GetSocietyRank] [Root.GetFirstName]\nThe recruit you sent us, [satanists_recruit.GetTitledFirstName], has proven to be disaster. We have spent significant resources in our attempt to induct [satanists_recruit.GetHerHim], all for naught. This lack of judgment will be remembered.\n\nSigned, [From.GetSocietyRank] [From.GetFirstName];;;;;;;;;;;;;x
+EVTDESC_MNM_4452;[From.GetSocietyGreeting], [Root.GetTitledFirstName]\n\nWe are delighted by your decision to join the ranks of [From.Society.GetName], [Root.GetTitle]. [From.GetSocietyGoal].\n\n[From.GetSocietyFarewell],\nSigned, [From.GetSocietyRank] [From.GetFirstName];;;;;;;;;;;;;x
+GetSocietyGreeting_cenarion_circle;Ishnu'alah;;;;;;;;;;;;;x
+GetSocietyGreeting_sisterhood_of_elune;Ishnu'alah;;;;;;;;;;;;;x
```

Comment:
> Ishnu'alah

What if CC/SoE leader culture is not Kaldorei?

<details>

![image](https://user-images.githubusercontent.com/46576860/64915565-af5f1b00-d772-11e9-8c12-fe038a5b56e4.png)

</details>

Reply:
Shame on those who do not know what this phrase means. It's absolute legend!

Reply:
Replaced...
**SAD**

Reply:
> Replaced...
**SAD**

You know, you can just add another Society Greeting if society leader is not kaldorei instead of replacing base greeting with generic

Reply:
No, Ishnu'alah is already dead.

Reply:
**SAD**